sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -904,10 +902,11 @@ public virtual Response GetSnapshot(string name, IEnumerable<string> select, Mat
         }
 
         /// <summary> Creates a configuration setting snapshot. </summary>
+        /// <param name="wait">TODO</param>
         /// <param name="name"> The name of the configuration setting snapshot to create. </param>
         /// <param name="snapshot"> The configuration setting snapshot to create. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<ConfigurationSettingsSnapshot>> CreateSnapshotAsync(string name, ConfigurationSettingsSnapshot snapshot, CancellationToken cancellationToken = default)
+        public virtual async Task<CreateSnapshotOperation> CreateSnapshotAsync(WaitUntil wait, string name, ConfigurationSettingsSnapshot snapshot, CancellationToken cancellationToken = default)
```

Comment:
Can we add the test cases to cover all the LRO scenarios for create operation?

Reply:
We initially discussed offline that I would not be writing the test cases or samples for the LRO case. But I added them per your feedback here.

Reply:
Yeah, that time we didn’t have dogfood resources working. Now that they are working, I thought since you have implemented the feature, you will have more context and if you add test cases, you will also be sure that your implementation is working.